reports-scheduler/src/main/java/com/amazon/opendistroforelasticsearch/reportsscheduler/rest/RestReportsJobAction.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.reportsscheduler.rest;
+
+import static com.amazon.opendistroforelasticsearch.reportsscheduler.common.Constants.BASE_SCHEDULER_URI;
+
+import java.util.List;
+
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.cluster.service.ClusterService;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.rest.BaseRestHandler;
+import org.elasticsearch.rest.BytesRestResponse;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.RestStatus;
+
+import com.amazon.opendistroforelasticsearch.reportsscheduler.rest.handler.ReportsJobActionHandler;
+import com.google.common.collect.ImmutableList;
+
+public class RestReportsJobAction extends BaseRestHandler {
+  public static final String SCHEDULER_JOB_ACTION = "reports_scheduler_job_action";
```

Comment:
np: consistency with camel case or snake case

Reply:
It's literally snake_case, it's the naming convention of constants.
> The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_"). (ANSI constants should be avoided, for ease of debugging.

Reply:
Sorry, I was referring to the `"reports_scheduler_job_action"` not the variable name

Reply:
changed "jobId" to "job_id" to use snake case in url 
https://github.com/elastic/elasticsearch/blob/20c4da3a64e3b1ccf5cef26e9e99b561df4efa76/plugins/examples/rest-handler/src/main/java/org/elasticsearch/example/resthandler/ExampleCatAction.java#L47